bindata/manifests/machine-config/units/ovs-configuration.service.yaml

```diff
@@ -0,0 +1,9 @@
+name: ovs-configuration.service
+dropins:
+- name: 10-switchdev.conf
+  contents: |
+    [Service]
+    ExecStartPre=-/sbin/ip route del default via 10.19.111.254 dev eno2
```

Comment:
Why this line is needed? It looks like a env specific config.

Reply:
it's for my test environment. will remove it.